solutions/standard/variables.tf

```diff
@@ -80,6 +80,24 @@ variable "public_engine_enabled" {
   default     = false
 }
 
+variable "default_secret_group_name" {
+  type        = string
+  description = "Name to give the secrets group automatically created when provisioning a new Secrets Manager instance."
+  default     = "default-group"
+}
+
+variable "default_access_group_name" {
+  type        = string
+  description = "Name to give the access group automatically created when provisioning a new Secrets Manager instance."
+  default     = "secrets_manager_group"
+}
+
+variable "access_group_ids" {
```

Comment:
misleading name here. Perhaps go with `access_group_users` or `access_group_user_ids` ?

Reply:
what happens if a user is added to the group outside of terraform (using accesshub for example). When terraform runs and finds an empty list, will it try to remove the users?